thorlcr/activities/lookupjoin/thlookupjoinslave.cpp

```diff
@@ -923,17 +938,17 @@ class CInMemJoinBase : public CSlaveActivity, public CThorDataLink, public CAllO
                         throw MakeActivityException(this, 0, "Out of memory: Unable to add any more rows to RHS");
 
                     rightSerializer->serialize(mbser, (const byte *)row.get());
-                    if (mb.length() >= MAX_SEND_SIZE || broadcaster.isStopping())
+                    if (mb.length() >= MAX_SEND_SIZE || broadcaster.stopRequested())
```

Comment:
I think this will no longer return true if setStopping() was called in start().  Is that correct?